lib/akamai_rspec/matchers/non_akamai.rb

```diff
@@ -42,8 +42,15 @@ module Matchers
         rescue RestClient::MaxRedirectsReached
           return true # Securely sent a redirect
         rescue Exception => e
-          @error = e
-          return false
+          begin
+            if e.response # Securely sent a non-success HTTP code
+                          # See: https://github.com/rest-client/rest-client/tree/1.8.x#exceptions-see-wwww3orgprotocolsrfc2616rfc2616-sec10html
```

Comment:
I'm not sure how to mock an invalid cert in the rspec testing ...  could be interesting ...